sdk/keyvault/azure-keyvault-keys/README.md

```diff
@@ -368,7 +368,7 @@ except ResourceNotFoundError as e:
 
 ### Logging
 This library uses the standard
-[logging](https://docs.python.org/3.5/library/logging.html) library for logging.
+[logging](https://docs.python.org/3.6/library/logging.html) library for logging.
```

Comment:
```suggestion
[logging](https://docs.python.org/3/library/logging.html) library for logging.
```

Reply:
The fleeting nature of a Python version's life is what makes it beautiful